src/lib/components/FormControls/FileInput/FileInput.tsx

```diff
@@ -1,32 +1,32 @@
 import classNames from 'classnames';
 import type { ComponentProps, ReactNode } from 'react';
 import { forwardRef } from 'react';
-import { excludeClassName } from '../../helpers/exclude';
-import { useTheme } from '../Flowbite/ThemeContext';
-import HelperText from './HelperText';
-import type { TextInputColors, TextInputSizes } from './TextInput';
+import { useTheme } from '../../Flowbite/ThemeContext';
+import HelperText from '../HelperText';
+import type { TextInputColors, TextInputSizes } from '../TextInput';
 
-export interface FileInputProps extends Omit<ComponentProps<'input'>, 'type' | 'ref' | 'color' | 'className'> {
+export interface FileInputProps extends Omit<ComponentProps<'input'>, 'type' | 'ref' | 'color'> {
   sizing?: keyof TextInputSizes;
   helperText?: ReactNode;
   color?: keyof TextInputColors;
+  inputClassName?: string;
```

Comment:
OK. This is not your fault, but this is a mess. I'm not sure about these inner `className` access, you know? But at the same time, in this case we are saying that the basic component is the `input`, but it is wrapped by a 3 `div`s.

Anyway, I'll advocate to the only apply the `className` to the component that holds the `theme.base`. So, please remove this one.